### PR TITLE
Implemented sanitizers for arrays and collections of logs and crlf ob…

### PR DESCRIPTION
**Pull request by BenShmuely**
Specifically of logs and crlf

### CODE REVIEW

File: src/main/java/io/whitesource/cure/Encoder.java [hunk 1]
```diff
@@ -68,17 +71,34 @@ public static String forOsCommand(final String param, char[] charsToIgnore) {
...
             .replaceAll(">", "&gt");
   }
 
+  /**
+   * Encoding content for logs.
+   *
+   * @param contents arrays {@link Object} contains all the contents.
+   * @return encoded log content.
+   */
+  public static Collection<?> forLogContent(final Collection<?> contents) {
```
Comment (nadavbarak) [inline src/main/java/io/whitesource/cure/Encoder.java:90]: why final? same for all functions

File: src/main/java/io/whitesource/cure/Encoder.java [hunk 2]
```diff
@@ -90,11 +110,47 @@ public static String forCrlf(final String content) {
       return null;
     }
     return StringUtils.replaceEach(
-            content.toString(),
+            formatToString(content),
```
Comment (nadavbarak) [inline src/main/java/io/whitesource/cure/Encoder.java:113]: content is already string

File: src/main/java/io/whitesource/cure/Encoder.java [hunk 3]
```diff
@@ -68,17 +71,34 @@ public static String forOsCommand(final String param, char[] charsToIgnore) {
...
    * @return encoded log content.
    */
-  public static String forLogContent(final Object content) {
+  public static String forLogContent(final String content) {
     if (content == null) {
       return null;
     }
-    return content
-            .toString()
+    return formatToString(content)
```
Comment (nadavbarak) [inline src/main/java/io/whitesource/cure/Encoder.java:78]: content is already string

File: src/main/java/io/whitesource/cure/Encoder.java [hunk 4]
```diff
@@ -57,7 +60,7 @@ public static String forOsCommand(final String param, char[] charsToIgnore) {
     List<String> results = new ArrayList<>();
```
Comment (nadavbarak) [inline src/main/java/io/whitesource/cure/Encoder.java:60]: Is this case not included in Collection<?> ?

**Review by nadavbarak — APPROVED** [review]
Fix based on comments and merge.
Can add one more test of something which is not null
